pkg/manifests/manifests.go

```diff
@@ -21,6 +22,13 @@ const (
 	DNSConfigMap          = "assets/dns/configmap.yaml"
 	DNSDaemonSet          = "assets/dns/daemonset.yaml"
 	DNSService            = "assets/dns/service.yaml"
+
+	OperatorCustomResourrceDefinition = "manifests/00-custom-resource-definition.yaml"
```

Comment:
Typo in name - it works but ... [OperatorCustomResour]`r`[ceDefinition]

Reply:
Fixed